src/components/VideoGrid/VideoGrid.test.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import VideoGrid from '.';
+import mockedData from '../../mockData/mockData.json';
+
+describe('Video Grid', () => {
+  const { items } = mockedData;
+
+  beforeEach(() => {
+    render(<VideoGrid videos={items} />);
+  });
+
+  it('renders all items', () => {
+    const grid = screen.getByTestId('grid');
+    expect(grid.children.length).toEqual(1);
+    const row = screen.getByTestId('row');
+    expect(row.children.length).toEqual(24);
```

Comment:
how about expecting the 24 elements are actually VideoCards? and that there's only one per Styled.Col